python/mxnet/contrib/onnx/mx2onnx/_op_translations.py

```diff
@@ -1840,7 +1840,7 @@ def convert_reshape(node, **kwargs):
     output_shape_list = convert_string_to_list(attrs["shape"])
 
     initializer = kwargs["initializer"]
-    output_shape_np = np.array(output_shape_list)
+    output_shape_np = np.array(output_shape_list, dtype='float32')
```

Comment:
don't use float for shape

Reply:
"shape" in reshape seems to be int64 according to ONNX spec. Changing it to int64

Reply:
@zhreshold @vandanavk  output_shape_np = np.array(output_shape_list, dtype='float32'). Is int64 or float32 or float64 right?